starter-code/public/javascript/index.js

```diff
@@ -1,23 +1,157 @@
 const charactersAPI = new APIHandler("http://localhost:8000")
 
 $(document).ready( () => {
+
+    // get all the characters in the database
+
   document.getElementById('fetch-all').onclick = function(){
+    fetchAll();
+  }
+
+  function fetchAll(){
+
```

Comment:
Add this line here so you don't add the same character every time you click fetch all 
```$("#charactersContainer").empty()```